c2corg_ui/static/js/auth/authservice.js

```diff
@@ -71,6 +71,30 @@ app.Authentication.prototype.isAuthenticated = function() {
   return !!this.userData && !this.isExpired_();
 };
 
+/**
+ * @return {number}
+ * @private
+ */
+app.Authentication.prototype.getUserId_ = function() {
```

Comment:
Do you really need this function? Why not simply calling ``this.userData['id']``?